tools/keyvaluestore.go

```diff
@@ -0,0 +1,201 @@
+package tools
```

Comment:
OK, can I call it 'keyvalue' for clarity? I even considered the option of making it a separate top-level package since it's not even specific to `git-lfs`...